configure

```diff
@@ -244,6 +244,8 @@ _has_posix_spawn=no
 _endian=unknown
 _need_memalign=yes
 _have_x86=no
+# Android NDK version
+_ndk="21.0.6113669"
```

Comment:
this is specified in build.gradle as well, please is it possible to have it in place only and the other will read it?

Reply:
Removed, now we look at the NDK source.properties and build.gradle and compare versions